cmd/editor_config.go

```diff
@@ -0,0 +1,153 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	u "github.com/cloudposse/atmos/pkg/utils"
+
+	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/config"
+	er "github.com/editorconfig-checker/editorconfig-checker/v3/pkg/error"
+	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/files"
+	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/utils"
+	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/validation"
+	"github.com/spf13/cobra"
+)
+
+var (
+	editorConfigVersion   = "v3.0.3"
+	defaultConfigFilePath = ".ecrc"
```

Comment:
Let's use the more conventional filename of .editorconfig

Reply:
Done